client/search-ui/src/input/SearchHistoryDropdown.tsx

```diff
@@ -42,6 +43,7 @@ interface SearchHistoryDropdownProps extends TelemetryProps {
 // button and the content
 const popoverPadding = createRectangle(0, 0, 0, 2)
 
+// eslint-disable-next-line react/display-name
 export const SearchHistoryDropdown: React.FunctionComponent<SearchHistoryDropdownProps> = React.memo(
     ({ recentSearches = [], onSelect, className, telemetryService }) => {
```

Comment:
You can remove the eslint disable by adding a name to the function: 
```suggestion
export const SearchHistoryDropdown: React.FunctionComponent<SearchHistoryDropdownProps> = React.memo(
    function SearchHistoryDropdown({ recentSearches = [], onSelect, className, telemetryService }) {
```

Reply:
Ah, makes sense, this should help with debugging later, thank you!